lib/xen_os.mli

```diff
@@ -47,10 +47,23 @@ module Memory : sig
   (** Memory allocation statistics. Units are the system word size, as used by
       the OCaml stdlib Gc module. *)
 
+  val stat: unit -> stat
+  (** [stat ()]  returns memory allocation statistics. This call is slower
+      than quick_stat, and the returned values are accurate. *)
+
   val quick_stat : unit -> stat
   (** [quick_stat ()] returns memory allocation statistics. This call is
       computationally cheap, but the returned values may not be completely
       accurate. *)
```

Comment:
This uses `malloc_footprint`, which is a precomputed value -- thus inaccurate but quick.